src/js/common/components/Modal.jsx

```diff
@@ -78,7 +78,7 @@ class Modal extends React.Component {
     }
 
     return (
-      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={`${id}-title`}>
+      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={id && title ? `${id}-title` : undefined}>
```

Comment:
Yeah, as long as it's not the string `'undefined'`.

Yeah, same. It's pretty hard to figure out what the right thing is without constant access to someone who knows how to use the main screen readers well.